server/graphql/v2/query/collection/AccountsQuery.ts

```diff
@@ -39,7 +39,7 @@ const AccountsQuery = {
     },
     supportedPaymentMethodService: {
       type: new GraphQLList(PaymentMethodService),
-      description: 'Only accounts that support one of these payment services will be returned',
+      description: '@deprecated: Introduced for Hacktoberfest and not used anymore.',
```

Comment:
What is this syntax for deprecation? I don't see it used anywhere.

Reply:
Ops, sorry, a mistake on my part; we need to use `deprecationReason` here. 🤯